src/main/java/com/training/services/ga/validate/RequestBody.java

```diff
@@ -5,4 +5,8 @@
 @Setter
 public class RequestBody {
     String email;
+    String vdsId;
+    String lastName;
+    String loyaltyId;
+    String birthdate;
 }
```

Comment:
Trying to understand the use of this pojo? Is this for validate or loyalty api?
And if its for validate - then what are the other variables doing here.
Please review your own code once - after you raise a PR/push the code